page-stuff.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function(){
+  $('#new_comment_button').on('click', function() {
```

Comment:
If the only thing you're going to do in this callback is call another function, you can actually just call the function on the click! It would look like:
`$('#new_comment_button').on('click', hideShowButtonForm)`